### PR TITLE
add face generation example using torchnet

### PR DESCRIPTION
**Pull request by qiuxin2012**

### CODE REVIEW

**Comment by jason-dai** [comment]
1) change to apps/pytorch
2) add a readme on where this notebook is migrated from

**Comment by qiuxin2012** [comment]
Update the notebook, the main change is run pytorch to generate 64 faces, then load the pytorch model to analytics-zoo and generate 64 faces on spark.

File: apps/pytorch/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+pip install analytics-zoo==0.6.0.dev4
+```
+
+## Run Jupyter
+If you want to run spark local, just start jupyter notebook:
+```
+jupyter notebook
+```
+
+If you want to run on a yarn cluster(yarn-client mode), export env `HADOOP_CONF_DIR` and `ZOO_CONDA_NAME` before starting jupyter notebook.
```
Comment (jason-dai) [inline apps/pytorch/README.md]: Do we need the python dependency on workers? If not, we don't need coda environment.

File: apps/pytorch/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+```
+export HADOOP_CONF_DIR=[path to your hadoop conf directory]
+export ZOO_CONDA_NAME=[conda name intalled zoo]
```
Comment (jason-dai) [inline apps/pytorch/README.md]: How does the user set up the above env variable? Please add appropriate guide or external reference.

File: apps/pytorch/README.md (hunk 1 above)
Comment (qiuxin2012) [reply apps/pytorch/README.md]: Yes, we use python to create data RDD.

**Pull request merged**